schema.json

```diff
@@ -0,0 +1,404 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
should the schema have an `$id`? It seems standard on other JSON schema documents.

Reply:
Also how about a `title` and a `description`?